web/src/test/java/org/cbioportal/web/util/ClinicalDataBinUtilTest.java

```diff
@@ -157,10 +187,6 @@ public void testUnfilteredFetchClinicalDataBinCounts() {
         // study view filter should be applied only once
         verify(studyViewFilterApplier, times(1)).apply(any());
         
-        // ids should be populated only once
-        verify(clinicalDataBinUtil, times(1))
-            .populateIdLists(any(), any(), any(), any(), any(), any(), any(), any(), any(), any(), any());
```

Comment:
Are we still keeping this method `populateIdLists`? If not, are we replacing it with a new function? If that's the case, maybe we should verify the new function.

Reply:
Fixed in [948c8b9](https://github.com/cBioPortal/cbioportal/pull/10039/commits/948c8b95483aa135b63bc27c1b3a3e7f886b08a3)